src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,7 +10,7 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated.
  */
-public class Person implements ReadOnlyPerson {
+public class Person implements ReadOnlyPerson, Comparable<Person> {
```

Comment:
Good way of implementing a natural ordering within the address book by inheriting from the Comparable class and implementing polymorphism.